### DISCUSSION TITLE
Next. Js still relies on node_modules after it is packaged

### DISCUSSION THREAD

**Discussion by goc-chushixian** [webpack/webpack#16141] [Q&A]
After I build, I see that the packaged file still depends on many dependency packages in node_module. What should I do to fully package the dependencies in node_module

**Comment by alexander-akait** [comment]
Please provide example of a problem

**Reply by goc-chushixian** [reply]
```   
config.optimization.splitChunks ={
      cacheGroups: {
        commons: {
          test: /[\\/]node_modules[\\/]/,
          name: 'vendors_csx',
          chunks: 'all',
        }
      }
    }
```

**Reply by goc-chushixian** [reply]
This is the Webpack configuration in my next. config.js. I find that after I build, I still need the next support in node_modules, otherwise my website won't work

**Reply by goc-chushixian** [reply]
What should I do to pack all my dependencies into one folder and get my Next project running

**Comment by vankop** [comment]
please provide webpack config. probably you have `externals` option
